hypothesis-python/tests/array_api/conftest.py

```diff
@@ -36,29 +36,29 @@
         try:
             xp = name_to_entry_point["numpy"].load()
             xps = make_strategies_namespace(xp)
-            params = [pytest.param(xp, xps, id="numpy")]
+            params = [pytest.param(xp, xps, id=f"numpy-{xps.api_version}")]
```

Comment:
Might explore `params` being a list of `(xp, xps)` tuple pairs, as opposed to pytest params, and then standardise how the ids are generated (in the metafuncy hook, in a final step, etc.).